### ISSUE TITLE
dex_imdb_wiki.caffemodel.pt

### ISSUE THREAD

**Issue by XIAOKANGWANDE666** [InterDigitalInc/HRFAE#12]
could you please provide the dex_imdb_wiki.caffemodel.pt?

**Comment by farhodfm** [comment]
Hi @XIAOKANGWANDE666 

You can easily generate this model. The authors provide nice instructions (Credits to authors). It takes a few minutes.

But if you can't do that due to some reasons, download via this [link](https://ufile.io/850com8n). The file will be stored for 30 days.

**Comment by XIAOKANGWANDE666** [comment]
Thanks you very much!

**Comment by farhodfm** [comment]
@XIAOKANGWANDE666, you are welcome!

If everything is fine, then you can close this issue to help authors.

**Comment by XIAOKANGWANDE666** [comment]
the dataset is so lange.Do you run the train code?I want to train at lower resolution picture,I don't know if it works?I will close it later.

**Comment by farhodfm** [comment]
@XIAOKANGWANDE666 

1. > the dataset is so lange.

I am sorry, what do you mean by 'lange'?

2. > Do you run the train code?

Yes, I ran it yesterday. I could train it, and see training results using tensorboard (saving to logs included in the code).

3.  > I want to train at lower resolution picture,I don't know if it works?

I used [FFHQ @256x256](https://www.kaggle.com/xhlulu/flickrfaceshq-dataset-nvidia-resized-256px) resolution for training, but I didn't fully train the network, just checked :) So, I am not sure whether it will perform well or worse. And you can see discussions related to this by checking other [issue](https://github.com/InterDigitalInc/HRFAE/issues/10). By doing so, you can find ou that the authors trained the network for three different resolutions: 256x, 512x, 1024x.

**Comment by XIAOKANGWANDE666** [comment]
thanks！

**Comment by farhodfm** [comment]
@XIAOKANGWANDE666, you are welcome! Good luck with your research

**Comment by XIAOKANGWANDE666** [comment]
excuse me？Do you train the code  only with 70,000 1024x1024 pictures?I see that FFHQ has 70,000pictures of 1024x1024,but auther use styleGAN create more.The resdme only train 70,000?Coulad you please tell me how you do it?

**Comment by farhodfm** [comment]
Hi, @XIAOKANGWANDE666 

Sorry for the late reply.

I used the original [FFHQ](https://drive.google.com/file/d/1WvlAIvuochQn_L_f9p3OdFdTiSLlnnhv/view?usp=sharing) with 70K images along with [labels](https://partage.imt.fr/index.php/s/DbSk4HzFkeCYXDt) provided by authors.

Yes, the authors used an augmented dataset, namely age-equalized, using StyleGAN (check Section 4.1 of the paper).
But unfortunately, I couldn't find any link to get this dataset. I even raised an issue [here](https://github.com/InterDigitalInc/HRFAE/issues/11).

So, basically, you can run their code using the original set which I am doing these days.
But, I am not sure about performance by training using this set.

**Comment by XIAOKANGWANDE666** [comment]
how much time do you spend on training?

**Comment by LHL921026** [comment]
> Hi @XIAOKANGWANDE666
> 
> You can easily generate this model. The authors provide nice instructions (Credits to authors). It takes a few minutes.
> 
> But if you can't do that due to some reasons, download via this [link](https://ufile.io/850com8n). The file will be stored for 30 days.

Hi, could you please provide the .pt file for me as well? I had trouble when converting the caffe model

**Comment by quleileileilei** [comment]
could you please provide the dex_imdb_wiki.caffemodel.pt?

**Comment by AOguo21** [comment]
> 你好
> 
> 您可以轻松生成此模型。作者提供了很好的说明（作者的功劳）。这需要几分钟。
> 
> 但是，如果由于某些原因无法执行此操作，请通过此[链接](https://ufile.io/850com8n)下载。该文件将存储 30 天。

could you please provide the dex_imdb_wiki.caffemodel.pt?
